gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data
+from ...utils.energy import EnergyBounds
+from ...data import DataStore, ObservationList
+
+
+@requires_data('gammapy-extra')
+def test_make_psftable():
```

Comment:
Can you please still put this test?
If there's an issue with our PSF, we'll fix it, we shouldn't ignore it.
You can add `pytest.mark.xfail` for now if it fails for an unrelated reason, so that this pull request can go in.

There's a few other inline comments from me that aren't addressed though, so this isn't ready to be merged yet.